setup.py

```diff
@@ -94,7 +94,7 @@ def get_git_sha() -> str:
         "graphlib-backport",
         "gunicorn>=20.1.0",
         "hashids>=1.3.1, <2",
-        "holidays==0.14.2",
+        "holidays>=0.14.2",
```

Comment:
While we're at it, should we bring this one up to the current version also?
```suggestion
        "holidays>=0.16.0, <0.17",
```
The changelog looks pretty straight forward, and I kind of like the added support for 3.11 in 0.15 which we'll soon start working towards: https://github.com/dr-prodigy/python-holidays/blob/master/CHANGES